user-guide/Standard_Apps/Monitoring_Solutions/Performance_Analyzer/Performance_Analyzer_LCA.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Performance_Analyzer_LCA
+---
+
+# Problem
```

Comment:
Could you change this page so it starts with a title that makes clear what the page is about ("Performance Analyzer low-code app"?), followed by a short introduction? Imagine you're a user who ends up on this page from the search: it has to be clear from the start what the rest of the page is about.

Reply:
Done